smart-contracts/contracts/test-artifacts/TestEventHooks.sol

```diff
@@ -78,4 +78,16 @@ contract TestEventHooks is ILockKeyPurchaseHook, ILockKeyCancelHook
   {
     emit OnKeyCancel(msg.sender, _operator, _to, _refund);
   }
+
+  
+  function hasValidKey(
+    address _lockAddress,
+    address _from,
+    uint _expirationtimeStamp
+  ) external view
+    returns (bool isValidKey)
+  { 
+    return _expirationtimeStamp < block.timestamp;
```

Comment:
That test case is a bit weak - it only uses one param. Not sure how to improve it, maybe by using some more conditions?

Reply:
Oh, maybe we can make it just return true for an hardcoded address that does not have a valid membership?